libs/sam-formly-sample/src/lib/feature/formly-sample/services/autocomplete-sample.service.ts

```diff
@@ -0,0 +1,68 @@
+/* tslint:disable */
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { SDSAutocomplteServiceInterface, SDSHiercarchicalServiceResult } from '@gsa-sam/components';
```

Comment:
Spelling issue:
SDSAutocomplteServiceInterface

Reply:
This is the component service by Thomas. Can we discuss in the parling lot.